lib/datadog/open_feature/transport/http.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative '../../core/transport/http'
+
+require_relative 'http/api'
+require_relative 'exposures'
+
+module Datadog
+  module OpenFeature
+    module Transport
+      # Namespace for HTTP transport components
+      module HTTP
+        module_function
+
+        def build(agent_settings:, logger: Datadog.logger, headers: nil)
```

Comment:
(Same note as in other default logger parameters)
```suggestion
        def build(agent_settings:, logger:, headers: nil)
```